config_neural_network_models.py

```diff
@@ -19,33 +19,33 @@
 #   - The loss function
 
 MultiLayer_Perceptron \
```

Comment:
PEP8 cautions against backslashes in Python code because a whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.  It would therefore be customary to change this to:
```suggestion
MultiLayer_Perceptron = [
```

Reply:
IntelliJ / PyCharm ist suggesting to break the code this way.

Reply:
PyCharm does not support black formatting?  https://github.com/psf

Reply:
Have not heard about black code formatter before. Sounds interesting.